packages/convai-widget-core/src/types/config.ts

```diff
@@ -40,12 +60,34 @@ export type AvatarConfig =
     };
 
 export const DefaultTextContents = {
```

Comment:
Wonder how are these going to be translated based on selected language. Are they?